### PR TITLE
fix: hasOwnProperty not found

### PR DESCRIPTION
**Pull request by gemwuu**
An `obj` created by `Object.create(null)` doesn't has `hasOwnProperty` method.

### CODE REVIEW

**Review by fengmk2 — APPROVED** [review]
LGTM

File: test/method-test.js [hunk 1]
```diff
@@ -79,10 +79,19 @@ describe('Mock property', () => {
     enableCache: true,
     delay: 10
   };
+
+  const plainObj = Object.create(null);
+  plainObj.testKey = 'testValue';
+
```
Comment (fent) [inline test/method-test.js]: place this inside of the `it`

File: package.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
   ],
-  "version": "2.0.0",
+  "version": "1.0.0",
   "repository": {
     "type": "git",
-    "url": "git://github.com/fent/muk-prop.js.git"
+    "url": "git://github.com/node-modules/muk-prop.js.git"
   },
   "author": "fent (https://github.com/fent)",
   "main": "./lib/index.js",
```
Comment (fent) [inline package.json]: please revert changes to this file

**Review by fent — CHANGES_REQUESTED** [review]
Thanks for your contribution! I left a couple of comments

**Comment by gemwuu** [comment]
> Thanks for your contribution! I left a couple of comments

It's my pleasure! This repo helps us a lot!

**Review by popomore — APPROVED** [review]
LGTM

**Review by fent — APPROVED** [review]

**Comment by popomore** [comment]
@fent It seem that we should release a major version, because you had upgrade node version in some commits.

**Comment by popomore** [comment]
I have released v3.0.0

**Comment by fent** [comment]
Thanks for that. Let me see if I can figure out how to list them on Releases

**Pull request merged**